homeassistant/components/tractive/device_tracker.py

```diff
@@ -101,6 +101,7 @@ def _handle_position_update(self, event: dict[str, Any]) -> None:
 
     async def async_added_to_hass(self) -> None:
         """Handle entity which will be added."""
+        await super().async_added_to_hass()
```

Comment:
I’m not able to test this change right now but I think this will break the device tracker platform.